src/main/java/io/r2dbc/postgresql/codec/DefaultCodecs.java

```diff
@@ -96,97 +96,6 @@ public DefaultCodecs(ByteBufAllocator byteBufAllocator, boolean preferAttachedBu
         this.codecLookup.afterCodecAdded();
     }
 
-    @SuppressWarnings({"unchecked", "rawtypes"})
```

Comment:
What's the reason for this change? It makes the pull request almost illegible because it mixes refactorings with actual changes.

Reply:
This looks to have been introduced as part of accidental reformatting and is an oversight on my part, I have created a new commit that rolls back the formatting changes to Default Codec.